buildkite/pipelines/gerrit-postsubmit.yml

```diff
@@ -7,6 +7,18 @@ platforms:
     - "--test_tag_filters=-slow"
     test_targets:
     - "..."
+  ubuntu1804_java9:
+    build_flags:
+    - "--host_java_toolchain=@bazel_tools//tools/jdk:toolchain_java9"
+    - "--java_toolchain=@bazel_tools//tools/jdk:toolchain_jdk9"
+    build_targets:
+    - "//:release"
+    test_flags:
+    - "--test_tag_filters=-slow"
```

Comment:
Can you also add here: `-flaky,-docker` here. This is done on our CI.

Reply:
Added (also to the other platforms)!